google/cloud/bigtable/data_client.cc

```diff
@@ -42,15 +44,15 @@ DataClient::PrepareAsyncSampleRowKeys(
  */
 class DefaultDataClient : public DataClient {
  private:
-  // Introduce an early `private:` section because this type is used to define
```

Comment:
See the other comment regarding why we document the deviation from the style guide.